templates/_helpers.tpl

```diff
@@ -76,6 +80,14 @@ Create the name of the service account to use
 {{- end }}
 {{- end }}
 
+{{- define "dictToList" -}}
+{{ $keyName := (get . "keyName" | default "name")}}
+{{- range $key, $value := .contents -}}
+- {{ $keyName }}: {{ $key }}
+{{- $value | toYaml | nindent 2 }}
+{{ end }}
```

Comment:
Shouldn't be to avoid a new line?
```suggestion
{{- end -}}
```

Reply:
no, otherwise the next item in the `range` is appended on the next item, newline needed in this case

Reply:
but I will double check what I say